source/module_basis/module_pw/fft.cpp

```diff
@@ -64,7 +64,7 @@ void FFT::clear()
 }
 
 void FFT::initfft(int nx_in, int ny_in, int nz_in, int lixy_in, int rixy_in, int ns_in, int nplane_in, int nproc_in,
-                  bool gamma_only_in, bool xprime_in, bool mpifft_in)
+                  bool gamma_only_in, bool xprime_in,bool mpifft_in,bool cuda_memory_allocate)
```

Comment:
change the variable name from "cuda_memory_allocate" to "gpu_flag"

Reply:
the problem has been finished